sonaranalyzer-dotnet/its/regression-test.ps1

```diff
@@ -83,7 +86,7 @@ function Build-Project-DotnetTool([string]$ProjectName, [string]$SolutionRelativ
         -flp:"logFile=output\${ProjectName}.log;verbosity=d" `
     } -errorMessage "ERROR: Build FAILED."
 
-    Remove-Item $globalJsonPath
+    Remove-Item $tempGlobalJsonPath
```

Comment:
If the PS script fails, we might end up with random file `\sonaranalyzer-dotnet\its\global.json`.

That should not be a problem, right? It should be deleted from build machine and locally we should notice that before commit. We might just wonder how it happened.

Reply:
Indeed. I can add the deletion in the `finally` block. Good point.